index.html

```diff
@@ -0,0 +1,83 @@
+
+              <div id="img-caption">
+                <p>
+                  Dr. Norman Borlaug, third from left, trains biologists in Mexico on how to i
+                  ncrease wheat yields - part of his life-long war on hunger.
+                </p>
+              </div>
+            </div>
+            <div id ="tribute-info">
+              <ul>
+                <li>
+                  <strong>1914</strong> - Born in Cresco, Iowa
+                </li>
+                <li>
+                  <strong>1933</strong> - Leaves his family's farm to attend the University of
+                  Minnesota, thanks to a Depression era program known as the "National Youth
+                  Administration"
+                </li>
+                <li>
+                  <strong>1935</strong> - Has to stop school and save up more money.
+                  Works in the Civilian Conservation Corps, helping starving Americans.
+                  "I saw how food changed them", he said. "All of this left scars on me."
+                </li>
+                <li>
+                  <strong>1944</strong> - Rejects a 100% salary increase from Dupont,
+                 leaves behind his pregnant wife, and flies to Mexico to head a new plant
+                 pathology program. Over the next 16 years, his team breeds 6,000 different
+                  strains of disease resistent wheat - including different varieties for
+                  each major climate on Earth.
+                </li>
+               <li>
+                 <strong>1945</strong> - Discovers a way to grown wheat twice each season,
+                doubling wheat yields
+              </li>
+              <li>
+                <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American
+               breed, creating a strain that responds well to fertilizer. It goes on to provide 95%
+                 of Mexico's wheat.
+               </li>
+              <li>
+                <strong>1962</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian
+               subcontinent in time to help mitigate mass starvation due to a rapidly
+             </li>
+            </ul>
+          </div>
+              <p>If you have time, you should read more about this incredible human
+            being on his<a id ="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">
+            wikipedia entry.</a>
+            </p>
+          </div>
+        </div>
+      </div>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
I would suggest you make use of [html semantic tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and read up on [html best pratices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).